lib/dmv.rb

```diff
@@ -9,8 +14,12 @@ def add_facility(facility)
   end
 
   def facilities_offering_service(service)
-    @facilities.find do |facility|
-      facility.services.include?(service)
-    end
+    selected_facility = []
+    @facilities.each do |facility|
+     if facility.services.include?(service)
+        selected_facility.push(facility)
+      end
+    end   
+    selected_facility
   end
```

Comment:
Nice work here, good debugging! You could also use the `find_all` or `select` method here. What you have is great, but those are enums that could slim up this code a bit. It would look something like: 

```ruby
  def facilities_offering_service(service)
    @facilities.find_all do |facility|
      facility.services.include?(service)
    end   
  end
```